sqlglot/helper.py

```diff
@@ -380,7 +380,9 @@ def is_iterable(value: t.Any) -> bool:
     Returns:
         A `bool` value indicating if it is an iterable.
     """
-    return hasattr(value, "__iter__") and not isinstance(value, (str, bytes))
+    from sqlglot import Expression
+
+    return hasattr(value, "__iter__") and not isinstance(value, (str, bytes, Expression))
 
```

Comment:
Not sure how expensive this import can be; it was the simplest solution, but perhaps a try-except block where we call `iter(value)` is faster / better?

Reply:
nah it's fine

Reply:
cool